### ISSUE TITLE
[BUG] html entities (&lt;) don't work after 'http://' 

### ISSUE THREAD

**Issue by ittayd** [Jermolene/TiddlyWiki5#5589]
Type (without quotes) '~http://&lt;foo' the expected result is 'http://<foo'. But 'http://&lt;foo' is shown adding a space anywhere in 'http://' solves the issue (but adds a space, which is not the intent)

**Comment by Jermolene** [comment]
Hi @ittayd all three examples in your post appear to have identical text:

<img width="935" alt="image" src="https://user-images.githubusercontent.com/174761/113848407-59a4f600-9790-11eb-9686-4adc2341aba1.png">

**Comment by Jermolene** [comment]
Do you mean that the problem is that `~http://&lt;foo` renders as `http://&lt;foo`, and you want it to render as `http://<foo`?

It sounds like you're documenting http routes. One technique could be:

```
\define route(path)
<$text text=<<__path__>>/>
\end

<<route "http://<foo>">>
```

By creating a simple macro you can then type the paths directly, without having to use entities to get the angle brackets in.

**Comment by ittayd** [comment]
Yes, I want to render as `http://<foo`. And I have just one such URL in the page, so it'll be awkward to define a macro just for it.

**Comment by Jermolene** [comment]
> Yes, I want to render as `http://<foo`. And I have just one such URL in the page, so it'll be awkward to define a macro just for it.

Then I'd suggest something like this:

<$text text="http://<foo>"/>
